app/services/out_of_band_session_accessor.rb

```diff
@@ -60,13 +60,21 @@ def put(data, expiration = 5.minutes)
       'warden.user.user.session' => data.to_h,
     }
 
-    session_store.
-      send(:set_session, {}, session_uuid, session_data, expire_after: expiration.to_i)
+    session_store.send(
+      :write_session,
+      {},
+      Rack::Session::SessionId.new(session_uuid),
+      session_data,
+      expire_after: expiration.to_i,
+    )
   end
 
   # @return [Hash]
   def session_data
-    @session_data ||= session_store.send(:load_session_from_redis, session_uuid) || {}
+    @session_data ||= session_store.send(
+      :find_session, {},
+      Rack::Session::SessionId.new(session_uuid)
+    ).last || {}
```

Comment:
I would use "all on same line" or "all on different lines" approach here
```suggestion
    @session_data ||= session_store.send(
      :find_session, {}, Rack::Session::SessionId.new(session_uuid),
    ).last || {}
```